recipe.md

```diff
@@ -13,7 +13,7 @@ _Ζύμη_
 _Γέμιση_
 
 * 4 κόκκινα μήλα μεσαίου μεγέθους
-* 1 κουτάλι σούπας κανέλα τριμμένη
+* 1 κουτάλι γλυκού κανέλα τριμμένη
```

Comment:
Thanks so much for fixing this. 😄 ⚡️ ✨ 

Θα μπορουσες να αλλάξεις το κουτάλι σε κουταλάκι; 🙇 🙏